Protobuild.toml

```diff
@@ -5,11 +5,6 @@ version = "unstable"
 # example that selects the ctrd vanity binary.
 # generator = "gogoctrd"
```

Comment:
According to https://grpc.io/docs/languages/go/quickstart/#regenerate-grpc-code, the new command line options for generating gRPC code is like

```
$ protoc --go_out=. --go_opt=paths=source_relative \
    --go-grpc_out=. --go-grpc_opt=paths=source_relative \
    helloworld/helloworld.proto
```

So we may need to have multiple generators instead.